index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
Yes, that is what i mean. I just always count it like this (and so do [SCSS-lint](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#nestingdepth), and the people mentioned above):
```
a { // 1
  & .foo { // 2
    &__foo { // 3
      & > .bar {} // 4
    }
  }
}
```

(and at-rules without any declarations are an exception, which you added with `at-rules-without-declaration-blocks`)